packages/element-library/src/audioSticker/test/output.js

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import AudioStickerOutput from '../output';
+
+describe('AudioSticker output', () => {
+  // eslint-disable-next-line jest/no-disabled-tests -- This test will fail until this PR - https://github.com/ampproject/amphtml/pull/39913 is merged.
+  it.skip('should produce valid AMP output', async () => {
```

Comment:
@Swanand01 The AMP validator should be updated now. Can you confirm whether this test passes now?

Reply:
It passed locally, now waiting for it to pass on this PR too.

Reply:
The extension 'amp-story-audio-sticker' is referenced at version 'undefined' which is an invalid version. Please use a valid version of this extension. (see https://amp.dev/documentation/components/amp-story-audio-sticker)

Reply:
@swissspidy How did we fix this the last time? I can't seem to remember 😅

Reply:
It first I thought it's weird that it passes locally but not on CI. It also did not pass for me locally either.

Then I realized the `@ampproject/toolbox-optimizer` package we use for the AMP validation tests has some heavy disk-based caching. So the information I had locally was super out of date.

86fc09aa5a168a1d4f6f8a4c33228fb4364c5a10 is an attempt at working around this by always using the latest version. Let's see if it works. 🤞